pytest_plugins/metadata_markers.py

```diff
@@ -7,9 +7,22 @@
 from robottelo.config import settings
 from robottelo.hosts import get_sat_rhel_version
 from robottelo.logging import collection_logger as logger
+from robottelo.utils.issue_handlers.jira import are_any_jira_open
 
 FMT_XUNIT_TIME = '%Y-%m-%dT%H:%M:%S'
 IMPORTANCE_LEVELS = []
+selected = []
+deselected = []
+
+
+def parse_comma_separated_list(option_value):
+    if isinstance(option_value, str):
+        if option_value.lower() == 'true':
+            return True
+        if option_value.lower() == 'false':
+            return False
+        return option_value.split(',')
```

Comment:
return [item.strip() for item in option_value.split(',')]

just to handle this case 
--blocked-by '123, 2333,  322'

Reply:
Updated the PR:)